### PR TITLE
Some more corrections for the release notes

### PR DESCRIPTION
**Pull request by victoryforce**

### CODE REVIEW

**Review by kmilos — COMMENTED** [review]
These (camera support section of [fff12b0](https://github.com/darktable-org/darktable/pull/15914/commits/fff12b019ae9c52ee186a3520dc2ec030c9e6500)) are autogenerated and denote a specific operation mode in RawSpeed as such, so probably shouldn't be changed.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -324,10 +324,10 @@ changes (where available).
 
 - Fixed focus distance detection for Nikon Z mount lens corrections.
 
-- Fixed incorrect cropping of sensor data for sRaw dng files.
+- Fixed incorrect cropping of sensor data for sRaw DNG files.
```
Comment (kmilos) [inline RELEASE_NOTES.md:327]:
```suggestion
- Fixed incorrect cropping of sensor data for LinearRaw DNG files.
```

**Comment by victoryforce** [comment]
> These (camera support section of [fff12b0](https://github.com/darktable-org/darktable/pull/15914/commits/fff12b019ae9c52ee186a3520dc2ec030c9e6500)) are autogenerated and denote a specific operation mode in RawSpeed as such, so probably shouldn't be changed.

Please clarify. I don't get your idea. If this section is autogenerated and cut-n-pasted into the release notes, are we not allowed to touch this text? Even if, for example, there is a typo or incorrect spelling? Why so?

Maybe I misunderstood you and you meant something else?

**Comment by kmilos** [comment]
> Please clarify. I don't get your idea.

There's just little point to it IMHO. It'll just get regenerated and rewritten the next time around.

**Comment by victoryforce** [comment]
> It'll just get regenerated and rewritten the next time around.

Sure! So, to fix this properly, we also need to fix the script that generates it. Otherwise, we will have to correct the text afterwards for each subsequent release notes.

Will you help with this?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -360,10 +360,10 @@ changes (where available).
   lighttable in this scenario has now been drastically improved and is
   fast and responsive even with ~50k images and 20 thumbnails per line.
 
-- Fixed pixel errors in RAW Chromatic Aberration and LMMSE demosaic
+- Fixed pixel errors in raw chromatic aberrations module and LMMSE demosaic
```
Comment (TurboGit) [inline RELEASE_NOTES.md:363]: I'll keep the capitalized version. This is the release notes not the lower case interface.

**Comment by TurboGit** [comment]
I have cherry-picked the 3 following commits:
- Fix the typo
- The correct spelling is Lightroom, not LightRoom
- Traditionally, 'b' stands for bits, 'B' for bytes (and we do mean bytes)

Not sure yet if I'll have more time before the launch.

**Comment by victoryforce** [comment]
> I have cherry-picked the 3 following commits:
> 
> * Fix the typo
> * The correct spelling is Lightroom, not LightRoom
> * Traditionally, 'b' stands for bits, 'B' for bytes (and we do mean bytes)
> 
> Not sure yet if I'll have more time before the launch.

At this point, I found at least two more typos.

File: RELEASE_NOTES.md (hunk 2 above)
Comment (victoryforce) [reply RELEASE_NOTES.md:363]: > I'll keep the capitalized version. This is the release notes not the lower case interface.

Then it should be 'Raw chromatic aberrations' and not the version in the text. And if we use the capitalized version here, we should do the same in other places where module names are mentioned. Like, 'RGB primaries' and 'Sigmoid'.

Comment (victoryforce) [reply RELEASE_NOTES.md:363]: We can choose any version, capitalized or not, but we must use it uniformly.

**Comment by TurboGit** [comment]
> At this point, I found at least two more typos.

Let's push the fix here and I'll integrate into the GitHub & dtorg message. Thanks.

**Comment by victoryforce** [comment]
> > At this point, I found at least two more typos.
> 
> Let's push the fix here and I'll integrate into the GitHub & dtorg message. Thanks.

OK.

**Comment by TurboGit** [comment]
@victoryforce : Thanks a lot, cherry picked:
- Fix another two typos

**Comment by kmilos** [comment]
> Sure! So, to fix this properly, we also need to fix the script that generates it. Otherwise, we will have to correct the text afterwards for each subsequent release notes.
> 
> Will you help with this?

I really wouldn't bother - those modes/variables are lowercase in RawSpeed source code, we just leave the status quo for these.

**Comment by victoryforce** [comment]
> > Sure! So, to fix this properly, we also need to fix the script that generates it. Otherwise, we will have to correct the text afterwards for each subsequent release notes.
> > Will you help with this?
> 
> I really wouldn't bother - those modes/variables are lowercase in RawSpeed source code,

The Rawspeed maintainer can spell them however he want, it doesn't have to dictate the spelling rules we use for darktable.

> we just leave the status quo for these.

What status quo are you talking about? For many releases, we've been correcting the spelling of at least a certain word types to what literate people expect. At the very least, you can't lowercase:
- initialisms
- proper names
- trademarks, names of companies, products, programs/libraries

I absolutely do not understand the purpose of raising the discussion at the last moment and casting doubt on the correctness of this fix.

**Comment by TurboGit** [comment]
> The Rawspeed maintainer can spell them however he want, it doesn't have to dictate the spelling rules we use for darktable.

To use our own spelling we need to fix `tools/check_camera_support`, the code generated in the last RELEASE_NOTES.md was done with:

`$ ./tools/check_camera_support --compare release-4.4.2 HEAD`

**Comment by victoryforce** [comment]
> > The Rawspeed maintainer can spell them however he want, it doesn't have to dictate the spelling rules we use for darktable.
> 
> To use our own spelling we need to fix `tools/check_camera_support`, the code generated in the last RELEASE_NOTES.md was done with:
> 
> `$ ./tools/check_camera_support --compare release-4.4.2 HEAD`

Yes, I've already written it down in my Todo.

**Pull request closed without merging**